### PR TITLE
[15.0][MIG] stock_move_line_auto_fill to v15.0

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
It works without any change. Supersedes stale PR here: https://github.com/OCA/stock-logistics-workflow/pull/969

@ForgeFlow

### CODE REVIEW

**Comment by sergio-teruel** [comment]
Hi @AaronHForgeFlow Can you review the tests please??

**Comment by sergio-teruel** [comment]
ping @AaronHForgeFlow

File: stock_move_line_auto_fill/tests/test_stock_picking_auto_fill.py [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+        )
+        return_pick.action_assign()
+        # return_pick.move_lines.quantity_done = qty
+        return_pick._action_done()
+        return return_pick
+
+    def test_return_twice(self):
+        # Covered case:
+        # Return more than one times not duplicate quant units in stock
+        self.picking_type_out.auto_fill_operation = True
```
Comment (sergio-teruel) [inline stock_move_line_auto_fill/tests/test_stock_picking_auto_fill.py:446]:
```suggestion
        self.picking_type_out.auto_fill_operation = True
        # By default in v15 there is an opertion type to return goods
        self.picking_type_out.return_picking_type_id.auto_fill_operation = True

```

Comment (sergio-teruel) [reply stock_move_line_auto_fill/tests/test_stock_picking_auto_fill.py:446]: To fix the test

**Comment by pedrobaeza** [comment]
CI is red

**Comment by pedrobaeza** [comment]
Superseded by #1141

**Pull request closed without merging**